ios/FluentUI/Navigation/Shy Header/ShyHeaderController.swift

```diff
@@ -210,6 +204,16 @@ class ShyHeaderController: UIViewController {
         view.bringSubviewToFront(paddingView)
     }
 
+    private func setupShyHeaderView() {
```

Comment:
why is this a neccesary chnage?

Reply:
Yep, it's not I was just cleaning up a bit by putting all the shy header setup in a function. Should I revert?